src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -77,6 +79,16 @@ function FormattedUrl({ url }: { url: string }) {
 }
 
 const DetailTable = ({ className, details }: { className?: string; details: Detail[] }) => {
+  if (details.length === 0) {
+    return (
+      <div className={classNames(className, "flex flex-col")}>
+        <div className={classNames(styles.row, "cursor-pointer py-1 hover:bg-toolbarBackground")}>
+          No entries
+        </div>
+      </div>
+    );
+  }
```

Comment:
This change may be debatable but it seemed nicer to have a fallback UI.

![Screen Shot 2022-04-19 at 4 15 27 PM](https://user-images.githubusercontent.com/29597/164088916-62d64748-b667-45ba-b767-7515435ab4da.png)

![Screen Shot 2022-04-19 at 4 15 34 PM](https://user-images.githubusercontent.com/29597/164088917-de37f621-f098-4d6d-9db5-2defc3307e79.png)